edit/edit.go

```diff
@@ -431,6 +436,27 @@ func sub1(ed *Editor, at addr, re *re1.Regexp, repl []rune) ([][2]int64, error)
 	return m, pend(ed, at, sliceSource(rs))
 }
 
+// nthMatch skips past the first n-1 matches of the regular expression
+func nthMatch(ed *Editor, at addr, re *re1.Regexp, n int) ([][2]int64, error) {
+	var m [][2]int64
```

Comment:
These are fine.
(I usually declare the error variable first. It doesn't matter, but I have this thing about liking it when the declarations are in increasing order of number of characters. Please don't make that change how you do things, but I thought it necessary to mention this quirk of mine.)